### ISSUE TITLE
Question: What is the "usual rotation metric"?

### ISSUE THREAD

**Issue by adosar** [naver/roma#20]
First of all, congratulations for this nice (and extremely lightweight) package!

I have a point cloud of shape `(N, 3)`, i.e. `xyz` coordinates of `N` points, and I would like to randomly rotate it.
It can be done as following:
```python
pcd = torch.randn(10, 3)  # Point cloud with 10 points.
rr = random_rotmat()  # Generate a single random rotation matrix.
new_pcd = pcd @ rr.T
```

However, I am not sure if this the proper way or I am missing something, since the docstring of `random_rotmat` says:

> Generates a batch of random 3x3 rotation matrices, uniformly sampled according to the **usual rotation metric**.

Can someone explain what this "usual rotation metric" means?

**Comment by rbregier** [comment]
>
> Hi, indeed this part of the doc can be confusing, and I'm open to
> suggestions.The sampling is uniform in the sense that the probability to
> draw a sample at a distance smaller than d from an arbitrary  rotation R
> should not depend on the choice of R, when considering for example the
> chordal or angular distances as metrics to measure distances. This is what
> most people would expect from a uniform sampling I guess.
>
I hope it helps.

**Comment by adosar** [comment]
@rbregier To get an intuition for the uniform sampling, I visualized 1000 rotations generated by `random_rotmat` with the following snippet:

```python
# Visualize RoMa rotations with Plotly.
point = torch.tensor([0., 0., 1.])
rot_points = [point @ random_rotmat().T for i in range(1_000)]  # Generate 1000 randomly rotated versions.
new_points = torch.stack(rot_points).numpy()

x, y, z = new_points[:, 0], new_points[:, 1], new_points[:,  2]
fig = go.Figure(data=[go.Scatter3d(x=x, y=y, z=z, mode='markers', marker=dict(size=1, opacity=0.5))])
fig.update_layout(margin=dict(t=0, b=0, l=0, r=0))
fig.show()
```
![image](https://github.com/user-attachments/assets/77f04c9e-68ad-434e-b498-36c0c09afa82)

From the visualization, I think this is the expected result.
